tests/check/check_h26xsigned_auth.c

```diff
@@ -1245,6 +1278,7 @@ signed_video_suite(void)
   tcase_add_loop_test(tc, no_signature, s, e);
   tcase_add_loop_test(tc, multislice_no_signature, s, e);
   tcase_add_loop_test(tc, recurrence, s, e);
+  tcase_add_loop_test(tc, detect_change_of_public_key, s, e);
```

Comment:
Put this in the same order as it appears in the code above, that is, right after camera_reset_on_signing_side